django/core/management/templates.py

```diff
@@ -98,10 +98,13 @@ def handle(self, app_or_project, name, target=None, **options):
         base_name = '%s_name' % app_or_project
         base_subdir = '%s_template' % app_or_project
         base_directory = '%s_directory' % app_or_project
+        camel_case_name = 'camel_case_%s_name' % app_or_project
+        camel_case_value = ''.join(x for x in name.title() if x not in ('-', '_'))
```

Comment:
I think dash can't be used in an app name, correct? e.g. "'foo-bar' is not a valid app name. Please use only numbers, letters and underscores."